packages/core/src/stdlib/modules/listModule.ts

```diff
@@ -303,6 +304,60 @@ export const listModule = InterpreterModule.create(
                     }
                 )
             ),
+            id(listProto).assignField(
+                "toString",
+                jsFun(
+                    (args, context) => {
+                        const self = args.getFieldValue(SemanticFieldNames.SELF, context);
+                        const length = assertNumber(self.getFieldValue(lengthField, context), "length field of a list");
+                        const maxDepthObject = args.getField("maxDepth", context).value;
+                        const maxDepth = maxDepthObject.isNull ? 4 : assertNumber(maxDepthObject);
```

Comment:
the docs says 3

Reply:
Yes, because I have to decrement it before recursing.
Otherwise it would be recursion depth `2`.

Reply:
ok, but it is a bit weird if the docs says "defaults to 3", but if I directly provide 3, something different happens?

Reply:
Makes sense. I'll adjust the docs.